packages/@icon-magic/icon-models/src/plugin-manager.ts

```diff
@@ -54,6 +52,58 @@ export async function applyPluginsOnAsset(
   return pluginResults[plugins.length - 1];
 }
 
+export async function applyPluginOnAssets(icon: Icon, plugin: BuildPlugin | GeneratePlugin, type: RegExp, hashing: boolean) {
```

Comment:
So do we still need the function above? `applyPluginsOnAsset`? I haven't see the rest of icon-magic to see if it is used but should it be removed?  Ty!

Reply:
So that is because of how the test is structured. If you look at `generate/index.js`, what's happening is that we're running generate with and without hashing off. And the way to check is by changing the source SVG to one with and without the widths and the flip class.

Reply:
Yes, it's part of build right now and I didn't touch any of those parts. Will have to do some cleanup later and change build to use the new function too. Created https://github.com/linkedin/icon-magic/issues/322 to track it. Thanks for the reminder!